server/src/actions.js

```diff
@@ -0,0 +1,166 @@
+const readNonEmptySnapshot = (snapshot) => {
+    const value = snapshot.val()
+    return value ? Promise.resolve(value) : Promise.reject('snapshot is empty')
+}
+
+module.exports = class Actions {
+
+    constructor(database) {
+        this.database = database
+    } 
+
+    incrementPluginInstanceCountForTopic(topicId) {
+        return this.database
+            .ref(`/v2/topics_index/${topicId}/plugin_instances_count`)
```

Comment:
wondering if we couldn't have this version passed in instead? that would make support for future versions a bit easier no? should we have an issue for it?

Reply:
I was thinking that as well, could easily become a class dependency orrrrrr we extract another collaborator for handling references (removing the path details)

I would prefer an issue to be raised than fix it in this PR, but I'll leave it up to you :dancer:

Reply:
happy with the issue (and the collaborator for that matter)

Reply:
tracked in #31